src/layerelement.cpp

```diff
@@ -704,9 +703,12 @@ int LayerElement::SetAlignmentPitchPos(FunctorParams *functorParams)
         Note *note = dynamic_cast<Note *>(this);
         assert(note);
         Chord *chord = note->IsChordTone();
-        int loc = PitchInterface::CalcLoc(note->GetPname(), note->GetOct(), layerY->GetClefLocOffset(layerElementY));
-        // Once we have AttLoc on Note
-        // if (note->HasLoc()) loc = note->GetLoc();
+        int loc = 0;
+        if (note->HasPname()) {
```

Comment:
For readability, I would prefer to have both `if` (this one and the one below) with our without `{}` but both the same. Small detail...